test/contrib/spark_test.py

```diff
@@ -118,11 +213,11 @@ def communicate(self):
             p = P()
             p.returncode = 1
             if stdout == subprocess.PIPE:
-                p.stdout = StringIO('stdout')
+                p.stdout = StringIO('stdout'.encode('utf-8'))
```

Comment:
I realized that I commented on the commit and not the PR so I just repeat.

It would be better to use bytes literal `b'stderr'` and to use directly `BytesIO` instead of redefining `StringIO`.